src/mappers/status-to-display-status.ts

```diff
@@ -0,0 +1,21 @@
+import { Status, DisplayStatus } from '../models/status';
+import { displayTime } from '../helpers/displayTime';
+
+export function statusToDisplayStatus(
+  status: Status,
+  realTime: Date,
+): DisplayStatus {
+  switch (status) {
```

Comment:
Don't worry too much about changing this now, but for future it might be nice to use an enum here